frontend_build/src/webpack.config.base.js

```diff
@@ -52,14 +53,13 @@ var cssLoader = {
 
 // for stylus blocks in vue files.
 // note: vue-style-loader includes postcss processing
```

Comment:
From a cursory inspection, it _is_ doing the auto-prefixing. However I don't understand where because previously `vue-style-loader` was the thing that triggered postcss